certbot/certbot/_internal/main.py

```diff
@@ -534,12 +534,20 @@ def _report_next_steps(config: configuration.NamespaceConfig, installer_err: Opt
 
     # If the installation or enhancement raised an error, show advice on trying again
     if installer_err:
-        steps.append(
-            "The certificate was saved, but could not be installed (installer: "
-            f"{config.installer}). After fixing the error shown below, try installing it again "
-            f"by running:\n  {cli.cli_command} install --cert-name "
-            f"{_cert_name_from_config_or_lineage(config, lineage)}"
-        )
+        # Special case where either --nginx or --apache were used, causing us to
+        # run the "installer" (i.e. reloading the nginx/apache config)
+        if config.verb == 'certonly':
+            steps.append(
+                f"The certificate was saved, but the plugin {config.installer} failed"
```

Comment:
This does get at the right spirit, but I think we can be even more clear. What about something like: 

`The certificate was saved, but was not picked up by the installer ({config.installer}) due to the installer failing to reload. After fixing the error shown below, try reloading {config.installer} manually.`

I don't love "picked up", but I'd like to try to somehow explain why it matters that the installer needs to be reloaded.

Reply:
maybe s/picked up/successfully loaded/?

Reply:
love it 🚢